src/components/WalletAssets.vue

```diff
@@ -209,13 +210,35 @@ export default class WalletAssets extends Mixins(LoadingMixin, FormattedAmountMi
       }
     }
     &-value {
+      height: 16px;
       font-size: var(--s-font-size-medium);
       font-weight: 800;
       letter-spacing: var(--s-letter-spacing-mini);
       line-height: var(--s-line-height-reset);
       .formatted-amount__decimal {
         font-weight: 600;
       }
+
+      @include formatted-amount-tooltip;
+    }
+  }
+
+  &__button.el-button.neumorphic.s-action:not(.s-primary).s-alternative {
```

Comment:
we don't have any specific type for these buttons right?

Reply:
Yes, this is usual action buttons, but according to mockup we need to override the styles of only these 3 wallet buttons